collector/manager/metrics_collector_manager.go

```diff
@@ -211,9 +224,9 @@ func (mcm *MetricsCollectorManager) declareMetricDefinitions(metricDetails []col
 
 		// put all the tags together for the full list of tags to be applied to this metric definition
 		allMetricTags := tags.Tags{}
-		allMetricTags.AppendTags(globalTags)   // global tags are overridden by...
+		allMetricTags.AppendTags(metricTags)   // metric tags are overriden by
 		allMetricTags.AppendTags(endpointTags) // endpoint tags which are overridden by...
-		allMetricTags.AppendTags(metricTags)   // metric tags.
+		allMetricTags.AppendTags(globalTags)   // global tags
```

Comment:
I think the metric ones should override the endpoint ones. Since the global ones are typically set by the admin, we can have those override the other two so global always "wins."